client/egypt-drug-checker-dashboard/src/components/header/coins/Coins.component.tsx

```diff
@@ -0,0 +1,26 @@
+"use client";
+import { Typography } from "@mui/material";
```

Comment:
use our convention MUtypography

Reply:
chore(frontend): adjusting name of the typography component to MUtypography